lib/bootstrap.js

```diff
@@ -46,6 +55,6 @@ async function runCLI(cli, commandIsUsed) {
 // eslint-disable-next-line space-before-function-paren
 (async () => {
```

Comment:
Not a real reason to use `async`/`await`, here.

Since you're not processing multiple promises, or catching an async error - you're not benefiting from `async`/`await`. Instead, I suggest:
```js
const commandIsUsed = isCommandUsed(commands);
const cli = new WebpackCLI();
runCLI(cli, commandIsUsed);
```

Which then lets you get rid of the IIFE `(async () => {})()`.